transcendence-app/src/shared/db/postgres/postgres.repository.ts

```diff
@@ -64,4 +65,27 @@ export class BasePostgresRepository<T> implements IBaseRepository<T> {
       ? entitiesData.map((entity) => new this.ctor(entity))
       : null;
   }
+
+  static insertWithClient<EntityData extends Record<string, any>>(
```

Comment:
suggestion:

```suggestion
  async insertWithClient<EntityData extends Record<string, any>>(
```

These methods will be inherited by children classes